Research/Research/ColorMatrix.json

```diff
@@ -399,4 +369,338 @@
         }]
     }
     ]
+},
+{
+    "version": 1,
+    "grayScale": {
+        "white": {
+            "color": "white",
+            "usesLightStyle": false
+        },
+        "veryLightGray": {
+            "color": "#EAEBEE",
+            "usesLightStyle": false
+        },
+        "lightGray": {
+            "color": "#C3C7D1",
+            "usesLightStyle": false
+        },
+        "gray": {
+            "color": "#9DA3B3",
+            "usesLightStyle": false
+        },
+        "darkGray": {
+            "color": "#777F95",
+            "usesLightStyle": true
+        },
+        "veryDarkGray": {
+            "color": "#575E71",
+            "usesLightStyle": true
+        },
+        "black": {
+            "color": "#2A2A2A",
```

Comment:
So, "black" is not actually black per se, just a quite dark gray? (not that any of the grays are actually a pure gray anyway)

Reply:
Yes, I have a question out to Stockard about the fact that black isn't black. I am going with the assumption for now that this was intentional. (Can update later if not)